packages/ffe-datepicker-react/src/datepicker/DatepickerComp.tsx

```diff
@@ -0,0 +1,356 @@
+                    min={1}
+                    max={31}
+                    onSpinButtonChange={(newValue, allowFocusNext = true) => {
+                        onChange(`${newValue}.${month}.${year}`);
+                        return allowFocusNext
+                            ? setDay(newValue, () =>
+                                  monthRef.current?.focus({
+                                      preventScroll: true,
+                                  }),
+                              )
+                            : setDay(newValue);
+                    }}
+                    nextSpinButton={monthRef}
+                    maxLength={2}
+                    aria-invalid={ariaInvalid()}
+                    aria-valuetext={`${day}`}
+                    aria-label={i18n[locale].DAY}
+                    aria-describedby={ariaDescribedby()}
+                    aria-labelledby={labelId}
+                >
+                    {day ? <PadZero value={day} /> : 'dd'}
+                </SpinButton>
+                .
+                <SpinButton
+                    ref={monthRef}
+                    value={month ?? undefined}
+                    min={1}
+                    max={12}
+                    onSpinButtonChange={(newValue, allowFocusNext = true) => {
+                        onChange(`${newValue}.${month}.${year}`);
+                        return allowFocusNext
+                            ? setMonth(newValue, () =>
+                                  yearRef.current?.focus({
+                                      preventScroll: true,
+                                  }),
+                              )
+                            : setMonth(newValue);
+                    }}
+                    nextSpinButton={yearRef}
+                    prevSpinButton={dayRef}
+                    maxLength={2}
+                    aria-invalid={ariaInvalid()}
+                    // @ts-ignore
+                    aria-valuetext={`${month} - ${i18n[locale][`MONTH_${month}`]}`}
+                    aria-label={i18n[locale].MONTH}
+                    aria-describedby={ariaDescribedby()}
+                    aria-labelledby={labelId}
+                >
+                    {month ? <PadZero value={month} /> : 'mm'}
+                </SpinButton>
+                .
+                <SpinButton
+                    ref={yearRef}
+                    className={'ffe-dateinput__field-year'}
+                    value={year ?? undefined}
+                    min={1}
+                    max={9999}
+                    onSpinButtonChange={newValue => {
+                        onChange(`${newValue}.${month}.${year}`);
+                        setYear(newValue);
+                    }}
+                    prevSpinButton={monthRef}
+                    maxLength={4}
+                    aria-invalid={ariaInvalid()}
+                    aria-valuetext={`${year}`}
+                    aria-valuenow={year ?? undefined}
```

Comment:
Hvis man skulle skriva tallet 0. Har sikkert ingen praktisk betyrning men kostar lite att fiksa
```suggestion
                    aria-valuenow={typeof year === 'number' ? year : undefined}
```